classes/class-backup.php

```diff
@@ -1370,7 +1370,7 @@ public function exclude_string( $context = 'zip' ) {
 				}
 
 				// Wrap directory fragments and files in wildcards for zip
-				if ( 'zip' === $context && ( $fragment || $file ) ) {
+				if ( 'zip' === $context && ( $fragment ) ) {
```

Comment:
I'd be tempted to remove this completely as the user does not know they are excluding all folders that match the pattern. They are just excluding a specific folder by clicking the Exclude button.